### ISSUE TITLE
Notes captured by users in the grid are wrapping and running down into multiple entries in the grid

### ISSUE THREAD

**Issue by malolaki** [openanthem/nimbus-core#520]
Notes captured by users in the grid are wrapping and running down into multiple entries in the grid

# Issue Details

**Type of Issue**  (check one with "X")
```
[X] Bug Report => Please search GitHub for a similar issue or PR before submitting
[ ] Feature Request => Please ensure feature is not already in progress
[ ] Support Request => Please do not submit support requests here, instead see: https://discourse.oss.antheminc.com/
```

## Current Behavior
When the users edit a line item in the grid and add a long note entry, the note gets wrapped and overlaps into the next line item in the grid making it Un-Readable.

## Expected Behavior
The note entries should not be wrapped and yet be fully readable on hovering the mouse pointer over the the note box.

### Steps to Reproduce
1. Login into the application
2. Edit one of the line items in the grid and add a long note and Save it
3. Check the note entry getting wrapped and overlapping into the next line items seen down below.

# Environment Details
* **Nimbus Version:** **1.3.1.M1**

* **Browser:** : **Google Chrome**

![image](https://user-images.githubusercontent.com/48562407/59786926-56c4d280-9296-11e9-97c5-789a68e05679.png)

**Comment by lopezton-anthem** [comment]
@malolaki, this may be resolved by the same resolution in #519. Please confirm.

If not resolved @malolaki, when you say to "edit" one of the line items will you please share how you're editing the value? I have written a `@Config` to manually update a value contained within a `@Grid` component and I see the `@Grid` component resizes the data fine.

Before:
![image](https://user-images.githubusercontent.com/29923536/59788131-b0c29980-9290-11e9-9797-03baf43b4085.png)

After:
![image](https://user-images.githubusercontent.com/29923536/59788160-c041e280-9290-11e9-946c-c0cfb90d501a.png)

**Comment by lopezton-anthem** [comment]
This issue is a duplicate of #519
